app/User.php

```diff
@@ -337,4 +342,121 @@ public static function getBlogModeratorEmails(): array
         return $emails;
     }
 
+    public function lastLogin()
+    {
+        return $this->hasOne(UserLogin::class)->latest();
+    }
+
+    public function isIdle(int $inactiveMonths = 1)
+    {
+
+        $now = new DateTime('NOW');
```

Comment:
do we need the `'NOW'` argument here?

Reply:
Removed 'NOW' argument as it is redundant. Resolved.